mantidproject/mantid
# Structured Detector Implementation

This represents the implementation of the StructuredDetector for irregular geometries #14965. 
### Testing

Code Review and run the LOKI IDF (you will need to change the file extension to .xml):
- Launch MantidPlot
- In Algorithms, type LoadEmptyInstrument then click execute
- Browse to the location of the LOKI IDF
- Select a workspace name e.g. "LOKI_Ws"
- Once the workspace has loaded, right click and select Show Instrument
- Ensure all views are correct and the instrument view works as expected with the new type of geometry.
#### LOKI IDF using structured detector

See attached
[LOKI_definition.txt](https://github.com/mantidproject/mantid/files/186328/LOKI_definition.txt)



Fixes #15542 .

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
